data-collection/test/utils.py

```diff
@@ -286,6 +288,7 @@ def _extract_lambda_arns(state):
     last_log_time = {lambda_arn: int(time.time()) * 1000 for lambda_arn in lambda_arns}
     execution_results = {execution_arn: None for execution_arn in execution_arns}
     running = True
+    time.sleep(20) # should refactor to base on waiting on crawlers
     while running:
```

Comment:
do we still need this sleep?

Reply:
As we add more modules, I am seeing a number of check fails only due to not enough time to catch up. There is throttling happening on the burst of crawler calls, which may be part of it. When there is time I think we should just rethink it a little to check the status.